apps/application-system/form/src/routes/Applications.tsx

```diff
@@ -14,24 +14,27 @@ import {
   ActionCard,
   Page,
   Button,
+  GridContainer,
 } from '@island.is/island-ui/core'
 import {
   Application,
   ApplicationStatus,
-  ApplicationTypes,
   coreMessages,
+  getTypeFromSlug,
 } from '@island.is/application/core'
 import { NotFound } from '@island.is/application/ui-shell'
 import { useApplicationNamespaces, useLocale } from '@island.is/localization'
+import { dateFormat } from '@island.is/shared/constants'
 
 import useAuth from '../hooks/useAuth'
 
 export const Applications: FC = () => {
-  const { type } = useParams<{ type: ApplicationTypes }>()
+  const { slug } = useParams<{ slug: string }>()
   const history = useHistory()
   const { userInfo } = useAuth()
   const { formatMessage } = useLocale()
   const nationalRegistryId = userInfo?.profile?.nationalId
+  const type = getTypeFromSlug(slug)
```

Comment:
`getTypeFromSlug` can return undefined, in that case we should probably have something like:
* `hasValidType = typeof type !== 'undefined'`
* pass `skip: !hasValidType` into `useQuery`
* `if (!hasValidType) return <NotFound />`